tests/test_contact_wrapper_classes.py

```diff
@@ -144,121 +99,91 @@ def test_external_contact_rod_rigid_body_without_collision():
 def test_external_contact_with_two_rods_with_collision():
 
     "Testing External Contact wrapper with two rods with analytical verified values"
+    "Test values have been copied from 'test_contact_specific_functions.py/test_calculate_contact_forces_rod_rod()'"
 
-    tol = 1e-6
     mock_rod_one = MockRod()
     mock_rod_two = MockRod()
-    mock_rod_two.position_collection = np.array(
-        [[1.2, 2.2, 3.2, 4.2, 5.2], [6, 7, 8, 9, 10], [11, 12, 13, 14, 15]]
-    )
-    ext_contact = ExternalContact(k=1.0, nu=0.5)
+    mock_rod_two.position_collection = np.array([[2.5, 3.5], [0, 0], [0, 0]])
+    ext_contact = ExternalContact(k=1.0, nu=0.0)
     ext_contact.apply_forces(mock_rod_one, 0, mock_rod_two, 0)
 
     assert_allclose(
         mock_rod_one.external_forces,
-        np.array(
-            [
-                [-5.848848, -7.696102, 2.569813, 0.257867, -0.091002],
-                [-3.177381, -2.169013, 2.890443, -0.240067, -0.864637],
-                [-1.309157, 4.732951, 7.427895, 0.333335, 0.835572],
-            ]
-        ),
-        rtol=tol,
-        atol=tol,
+        np.array([[-0.33, -0.66], [0, 0], [0, 0]]),
+        atol=1e-2,
     )
-
     assert_allclose(
         mock_rod_two.external_forces,
-        np.array(
-            [
-                [-6.167549, -1.298545, 10.66082, 1.435384, 0.030468],
-                [-2.17321, -4.425968, -1.173107, -0.060764, -0.887081],
-                [-4.672525, -7.453212, 2.279982, 0.528223, 0.718947],
-            ]
-        ),
-        rtol=tol,
-        atol=tol,
+        np.array([[0.33, 0.66], [0, 0], [0, 0]]),
+        atol=1e-2,
     )
 
 
 def test_external_contact_with_two_rods_without_collision():
 
     "Testing External Contact wrapper with two rods with analytical verified values"
 
-    tol = 1e-6
     mock_rod_one = MockRod()
     mock_rod_two = MockRod()
-    mock_rod_two.position_collection = np.array(
-        [
-            [100, 200, 300, 400, 500],
-            [600, 700, 800, 900, 100],
-            [110, 120, 130, 140, 150],
-        ]
-    )
-    ext_contact = ExternalContact(k=1.0, nu=0.5)
+
+    "Setting rod two position such that there is no collision"
+    mock_rod_two.position_collection = np.array([[100, 200], [600, 700], [110, 120]])
```

Comment:
please adjust the size of this as @armantekinalp said here too, also if you are making this change here in position collection you should also update rod two's lengths to match the new spacing between the nodes in position collection. Alternatively you could change the position collection to make the lengths stay the same.